content/how-to/upgrade-from-webpack-1.md

```diff
@@ -182,3 +182,60 @@ To keep compatiblity with old loaders, this information can be passed via plugin
 +   })
   ]
 ```
+
+
+#### Code Splitting with ES6
+
+The ES6 Loader spec defines `System.import` as method to load ES6 Modules dynamically on runtime.
+
+Webpack threads `System.import` as splitpoint and puts the requested module in a separate chunk.
+
+`System.import` takes the module name as argument and returns a Promise.
+
+``` js
+function onClick() {
+	System.import("./module").then(module => {
+		module.default;
```

Comment:
My first question as someone who has a project in webpack v1 with a lot of `require.ensure` calls: what does this line mean exactly? Maybe compare this to the old way, that would make it clear immediately.